posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -452,26 +458,75 @@ def _order_by(self, columns: list[str]) -> list[ast.OrderExpr] | None:
             elif field == WebAnalyticsOrderByFields.ERRORS:
                 column = "context.columns.errors"
 
+        def f(c: str) -> Optional[ast.OrderExpr]:
+            return ast.OrderExpr(expr=ast.Field(chain=[c]), order=direction) if column != c and c in columns else None
+
         return [
             expr
             for expr in [
+                # use order from query
                 ast.OrderExpr(expr=ast.Field(chain=[column]), order=direction)
                 if column is not None and column in columns
                 else None,
-                ast.OrderExpr(expr=ast.Field(chain=["context.columns.visitors"]), order=direction)
-                if column != "context.columns.visitors"
-                else None,
-                ast.OrderExpr(expr=ast.Field(chain=["context.columns.views"]), order=direction)
-                if column != "context.columns.views" and "context.columns.views" in columns
-                else None,
-                ast.OrderExpr(expr=ast.Field(chain=["context.columns.total_conversions"]), order=direction)
-                if column != "context.columns.total_conversions" and "context.columns.total_conversions" in columns
-                else None,
+                f("context.columns.unique_conversions"),
```

Comment:
I changed the order here because IMO the conversions one row should be sorted by conversion first